apps/src/music/views/InstrumentGrid/index.tsx

```diff
@@ -146,7 +163,7 @@ const InstrumentGrid: React.FunctionComponent<Props> = ({
             styles.innerCell
           )}
         >
-          {label}
+          {label.replace('#', '♯')}
```

Comment:
Thoughts on using the unicode sharp label in the actual key labels? Curious how this would look on the existing play notes block.

Reply:
Discussed this in person - looks like we do need the note names to use the hash symbol (`#`) since that's what ToneJS uses internally to look up the sample.